posthog/api/instance_status.py

```diff
@@ -141,6 +141,14 @@ def list(self, request: Request) -> Response:
 
         return Response({"results": {"overview": metrics, "internal_metrics": get_internal_metrics_dashboards()}})
 
+    # Used to capture internal metrics shown on dashboards
+    def create(self, request: Request) -> Response:
+        from posthog.internal_metrics import incr, timing
+
+        method = timing if request.data["method"] == "timing" else incr
```

Comment:
would it make sense to be a bit more defensive here & check that the method is actually "incr" in the else case and return some other response for unexpected methods?

Reply:
We could and it would be 'cleaner' that way - but it would be also extra complexity for an internal API used in one callsite. I'd rather not speculate on future use cases and refactor as needed on a small function.